airflow/providers/google/cloud/hooks/vertex_ai/supervised_fine_tuning.py

```diff
@@ -0,0 +1,101 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""This module contains a Google Cloud Vertex AI Supervised Fine Tuning hook."""
+
+from __future__ import annotations
+
+import time
+from typing import TYPE_CHECKING, Sequence
+
+import vertexai
+from vertexai.preview.tuning import sft
+
+if TYPE_CHECKING:
+    from vertexai.preview.tuning.sft import SupervisedTuningJob
+
+
+from airflow.providers.google.common.hooks.base_google import PROVIDE_PROJECT_ID, GoogleBaseHook
+
+
+class SupervisedFineTuningHook(GoogleBaseHook):
+    """Hook for Google Cloud Vertex AI Supervised Fine Tuning APIs."""
+
+    def __init__(
+        self,
+        gcp_conn_id: str = "google_cloud_default",
+        impersonation_chain: str | Sequence[str] | None = None,
+        **kwargs,
+    ):
+        if kwargs.get("delegate_to") is not None:
```

Comment:
I think this deprecation warning is not needed for the new Hook. Could you delete it?

Reply:
Done!